MPIDriver.py

```diff
@@ -76,7 +76,7 @@
         backend = 'theano'
         device = get_device( comm, args.masters, gpu_limit=args.max_gpus,
                 gpu_for_master=args.master_gpu)
-        print "Process",comm.Get_rank(),"using device",device
+        print ("Process",comm.Get_rank(),"using device",device)
```

Comment:
In Python 2 this syntax prints a tuple like ('Process', 0, 'using device', 'cpu').  These should be changed to use format strings